examples/rac-tailwind/src/index.js

```diff
@@ -3,3 +3,10 @@ import { App } from './App';
 
 let root = createRoot(document.getElementById('root'));
 root.render(<App />);
+
+// TODO: check on react-dom's version didn't work, figure out later
+// The below is for when react 17 is installed
+// import ReactDOM from 'react-dom';
+// ReactDOM.render(
+//   <App />, document.getElementById("root")
+// )
```

Comment:
Was doing a ReactDOM version check like in the other apps to decide which ReactDOM render call to use but it broke here in the tailwind app. Manually checked that our latest code on main worked in react 17 here, will followup on making this check more automatic